WebRoot/css/zm.css

```diff
@@ -2176,7 +2168,42 @@ div.CompDragArea * {
 }
 
 .ZmTaskEditView {
-	@PageBg@
+	@TaskEditView@
+}
+
+.ZmTaskEditView .ZmTaskEditHeader {
+	@TaskEditHeader@
+}
+
+.ZmTaskEditView .ZmTaskEditContent {
+	@TaskEditContent@
+}
+
+.ZmTaskEditHeader .ZmTaskEditHeaderTable {
+	@TaskEditHeaderTable@
+}
+
+.ZmTaskEditHeader .ZmTaskEditHeaderTable > tbody > tr > td {
+	@TaskEditHeaderTableCell@
+}
+
+.ZmTaskEditHeaderTable .ZmFieldsetContent td,
+.ZmTaskEditHeaderTable .ZmFieldsetContent div,
+.ZmTaskEditHeaderTable .ZmFieldsetContent label {
+	@TaskEditViewFontSize@
```

Comment:
Here is the one of the rule that specifies font-size of each elements,  https://github.com/Zimbra/zm-web-client/blob/develop/WebRoot/css/common.css#L86. 
(Which is very bad IMO)

So to override that, we have to have specific rule for each elements as well.

Let me know if you have any questions.